WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -80,7 +80,8 @@ private async Task<RoundState> WaitForRoundAsync(CancellationToken token)
 					&& roundState.CoinjoinState.Parameters.AllowedOutputAmounts.Min < MinimumOutputAmountSanity
 					&& roundState.Phase == Phase.InputRegistration
 					&& roundState.BlameOf == uint256.Zero
-					&& IsRoundEconomic(roundState.FeeRate),
+					&& IsRoundEconomic(roundState.FeeRate)
+					&& roundState.MaxSuggestedAmount > targetMaxSuggestedAmount,
```

Comment:
Given the base amount is std denomination and that it is expected the user's wallet will have utxos with std denominations i would suggest to use >= here

Reply:
ACK